data/json/items/tools.json

```diff
@@ -8589,10 +8589,10 @@
     "color": "dark_gray",
     "name": "survival knife",
     "name_plural": "survival knives",
-    "description": "This massive combat knife features a hollow handle with a compass built into the pommel and a row of fearsome looking saw teeth along the back of its blade.",
+    "description": "This massive combat knife features a hollow handle with a compass built into the pommel and a row of fearsome looking saw teeth along the back of its blade.  Being a Bowie knife on steroids, it is too large for most sheaths.",
```

Comment:
That "combat knife" part could be changed too. It is quite poor when it comes to actually killing stuff.